db/schema.rb

```diff
@@ -207,10 +207,10 @@
     t.text "cancellation_reason_comment"
     t.integer "nomis_event_ids", default: [], null: false, array: true
     t.uuid "profile_id"
-    t.boolean "move_agreed", default: false, null: false
-    t.string "move_agreed_by"
```

Comment:
I think those changes have been committed by mistake?

Reply:
indeed, when I recreate the DB, for some reasons, it recreates the schema that is  slightly different